{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -58,6 +62,7 @@ polint = "^0.4"
 dennis = "^1.1"
 dump-env = "^1.3"
 ipython = "^8.4"
+celery = {extras = ["redis"], version = "^5.2.7"}
```

Comment:
Duplicates runtime dep.
And it is better to pin `redis` too. Don't use extras when possible.